`3.39.0` changelog (#2471)
hashicorp/terraform-provider-google-beta: CHANGELOG.md

## 3.39.0 (Unreleased)

IMPROVEMENTS:
* compute: added network field to `compute_target_instance` ([#2456](https://github.com/hashicorp/terraform-provider-google-beta/pull/2456))
* compute: added storage_locations field to `google_compute_snapshot` ([#2461](https://github.com/hashicorp/terraform-provider-google-beta/pull/2461))
* container: Added `self_link` to google_container_cluster ([#2457](https://github.com/hashicorp/terraform-provider-google-beta/pull/2457))

> * container: marked `workload_metadata_config` as GA in `google_container_node_pool` ([#2458](https://github.com/hashicorp/terraform-provider-google-beta/pull/2458))

Reviewer: This can probably be removed

Author: 🤦‍♀️  I thought about removing it and apparently that convinced me I'd already done it.

BUG FIXES:

> * bigquery: fixed a bug when a big query table schema didn't have `name` in the schema it'd panic, now logs an error. ([#2462](https://github.com/hashicorp/terraform-provider-google-beta/pull/2462))

Reviewer: ```suggestion
* bigquery: fixed a bug when a BigQuery table schema didn't have `name` in the schema. Previously it would panic; now it logs an error. ([#2462](https://github.com/hashicorp/terraform-provider-google-beta/pull/2462))
```

## 3.39.0 (Unreleased)

IMPROVEMENTS:
* compute: added network field to `compute_target_instance` ([#2456](https://github.com/hashicorp/terraform-provider-google-beta/pull/2456))
* compute: added storage_locations field to `google_compute_snapshot` ([#2461](https://github.com/hashicorp/terraform-provider-google-beta/pull/2461))
* compute: added `kms_key_service_account`, `kms_key_self_link ` fields to `snapshot_encryption_key` field in `google_compute_snapshot`

> * compute: added `source_disk_encryption_key.kms_key_service_account` field to `google_compute_snapshot`

Reviewer 2: These two should have the same PR link as the top one I believe.

Author: Ah yep- I'd added the links to the original PR then copied them here. added in both tpg/tpgb.
